### PR TITLE
release: L1 to L2

### PR DESCRIPTION
**Pull request by Tulun**
Release of L1 transfers.

### CODE REVIEW

**Comment by daywiss** [comment]
getting an app crash: 
image

**Comment by daywiss** [comment]
- load app, be on bridge tab, be on optimism, have optimism selected in from dropdown
- select usdc asset, hit max
- switch to arbtrum in from dropdown, switch networks
- see error
image

**Comment by daywiss** [comment]
- reload page, bridge tab
- connect to optimism
- see that no tokens have balances
- select arbitrum in from, switch back to optimism in from
- see that token balances have loaded

**Comment by daywiss** [comment]
- connect with optimism and from optimism
- select asset uma
- change from to boba
- type number in input box in asset
- see app crash

**Comment by daywiss** [comment]
Theres an issue where if you have an invalid token selected in your asset dropdown and you switch from/to to a chain that doesnt support it, it does not get cleared, giving the impression its a valid asset to transfer. 
- select ethereum from
- select uma
- select boba to
- see that uma stays selected even though its not on boba

File: src/components/AddressSelection/AddressSelection.tsx [hunk 1]
```diff
@@ -124,7 +123,8 @@ const AddressSelection: React.FC = () => {
                   return (
                     <Item
                       className={
-                        t === sendState.currentlySelectedToChain
+                        t === sendState.currentlySelectedToChain ||
+                        t.chainId === ChainId.MAINNET
```
[the remark this answers is not in the document]
Comment (amateima) [reply src/components/AddressSelection/AddressSelection.tsx:127]: Yes, disable MAINNET destination when sending from MAINNET

File: src/components/ChainSelection/ChainSelection.tsx [hunk 2]
```diff
@@ -58,7 +55,7 @@ const ChainSelection: React.FC = () => {
     getItemProps,
     getMenuProps,
   } = useSelect({
-    items: filteredChains,
+    items: CHAINS_SELECTION,
```
[the remark this answers is not in the document]
Comment (amateima) [reply src/components/ChainSelection/ChainSelection.tsx:58]: Yes, we no longer need to filter chains since we support all of them

**Review by amateima — APPROVED** [review]

**Comment by amateima** [comment]
looking now on the mentioned bugs

**Comment by Tulun** [comment]
> * load app, be on bridge tab, be on optimism, have optimism selected in from dropdown
> * select usdc asset, hit max
> * switch to arbtrum in from dropdown, switch networks
> * usdc selected, hit max
> * see error
>   image
> * select eth asset, hit max
> * select uma asset, hit max
> * switch back to optimism
> * see app crash with same error

Believe I am fixed this with my PR.

**Pull request merged**